### ISSUE TITLE
Sound block enhancement : sound preview

### ISSUE THREAD

**Issue by Vincebecker** [aseba-community/thymio-vpl2#118]
It could be nice to have a "play" button when you are creating a sound in VPL mobile.
So you can hear a preview of the sound before sending it to the robot.

The sound could come directly from the tablet/mobile phone.

_Illustration of the idea :_
![screenshot_20170914-120752](https://user-images.githubusercontent.com/10742038/30424364-0603c61c-9946-11e7-85f9-2c6ecf6c560f.jpg)

**Comment by stephanemagnenat** [comment]
I agree, it is a good idea! In general, we should have more interaction between the block editors and the robot.

**Comment by Vincebecker** [comment]
>  I think that it would be better to produce the sound on the robot, because we want the users to focus on programming the robot in a tangible way.


I thought it was useful to produce the sound on the tablet/phone in the following cases : 

- robot is already doing something and maybe we don't want to interupt
- robot is far and you can't hear correctly the sound

But this is not so important. I will trust your judgment.

**Comment by stephanemagnenat** [comment]
I am not totally against previewing the sound on the tablet, but one problem we observed is that the link between the tablet and the robot might be non-trivial to make, so if we preview the sound on the robot it might help. But for sure this needs testing.
